tests/integration/Application/ViewObject/EntityActionsTest.php

```diff
@@ -48,18 +48,14 @@ public function test_oidc_entities_can_reset_client_secret($expectation, $protoc
     public static function resetClientOptions()
     {
         return [
-            [true, Entity::TYPE_OPENID_CONNECT, Entity::STATE_PUBLISHED, 'Published OIDC entity should have reset option'],
```

Comment:
Why not leave them and test the opposite of the happy flow?

Reply:
My removing them was anticipating on the timely removal of the type. But I think leaving them in place is valuable for now. So I reinstated the tests. Also added additional tests to verify the read only feature works correctly.